searx/settings.yml

```diff
@@ -33,7 +33,7 @@ brand:
 search:
   # Filter results. 0: None, 1: Moderate, 2: Strict
   safe_search: 0
-  # Existing autocomplete backends: "baidu", "brave", "dbpedia", "duckduckgo", "google", "yandex",
+  # Existing autocomplete backends: "360so", "baidu", "brave", "dbpedia", "duckduckgo", "google", "yandex",
```

Comment:
Please also add it to https://github.com/searxng/searxng/blob/master/docs/admin/settings/settings_search.rst.